core/frontend/src/NativeApp.ts

```diff
@@ -66,6 +66,7 @@ export class NativeApp {
 
   /** A Proxy to call one of the [NativeAppFunctions]($common) functions via IPC. */
   public static nativeAppIpc = IpcApp.makeIpcProxy<NativeAppFunctions>(nativeAppIpcStrings.channelName);
+  public static catalogIpc = IpcApp.makeIpcProxy<CatalogIModel.IpcMethods, CatalogIModel.IpcChannel>("catalogIModel/ipc");
```

Comment:
should prob be `CatalogIModel.IpcChannel` from common

Reply:
`CatalogIModel.IpcChannel` is a type, not a string. It is used as the second argument to `makeIpcProxy`. This is a new approach to avoid adding objects to the common package.